### PR TITLE
Updated giphy Socket

### PR DESCRIPTION
**Pull request by mattlsp**

### CODE REVIEW

File: .dist/package.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
+{
+  "name": "@eyedea-sockets/giphy",
+  "version": "0.0.1",
+  "description": "Description",
```
Comment (mkucharz) [inline .dist/package.json]: Missing description.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Syncano Socket for getting random giphy content by tag
+
+[![Syncano Socket](https://img.shields.io/badge/syncano-socket-blue.svg)](https://syncano.io)
+[![CircleCI branch](https://img.shields.io/circleci/project/github/eyedea-io/syncano-socket-document-generator/master.svg)](https://circleci.com/gh/eyedea-io/syncano-socket-document-generator/tree/master)
```
Comment (mkucharz) [inline README.md]: `document-generator`?

File: coverage/clover.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mkucharz) [inline coverage/clover.xml]: Remove whole `coverage` folder and make sure that it is in `.gitignore' (add `.gitignore` if missing).

File: socket.yml [hunk 4]
```diff
@@ -1,54 +1,10 @@
...
-          example: |
-            {
-              "message": "Something went wrong!"
-            }
+    description: Endpoint for getting giphy content by tag
+    inputs:
+      properties:
+        name:
+          type: string
+          description: giphy tag
```
Comment (mkucharz) [inline socket.yml]: Why not complete definition?

File: socket.yml [hunk 5]
```diff
@@ -1,54 +1,10 @@
-name: giphy
-description: Integration with Giphy (giphy.com)
-version: 0.0.4
-keywords:
```
Comment (mkucharz) [inline socket.yml:4]: Move those keywords to `package.json`

File: test/__mocks__/axios.js [hunk 6]
```diff
@@ -0,0 +1,2 @@
+import axios from 'axios'
```
Comment (mkucharz) [inline test/__mocks__/axios.js]: This mock is not used. Delete it.

File: test/__mocks__/fb.js [hunk 7]
```diff
@@ -0,0 +1,2 @@
+import fb from 'fb'
```
Comment (mkucharz) [inline test/__mocks__/fb.js]: This mock is not used. Delete it.

**Comment by mkucharz** [comment]
Remove whole `.dist` and add it to `.gitignore`.

File: test/unit/endpoint.test.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
+/* global describe it */
```
Comment (mkucharz) [inline test/unit/endpoint.test.js]: Rename filename to match source file.

File: test/unit/endpoint.test.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+import {run} from '@syncano/test'
+
+describe('generate', function () {
```
Comment (mkucharz) [inline test/unit/endpoint.test.js]: Change from `generate` from `random`.

**Review by mkucharz — CHANGES_REQUESTED** [review]

**Comment by mattlsp** [comment]
Changes added.

**Review by mkucharz — APPROVED** [review]

**Pull request merged**